api/models/usersDirectMessageThreads.js

```diff
@@ -92,6 +92,49 @@ const updateDirectMessageThreadNotificationStatusForUser = (threadId: string, us
 ===========================================================
 */
 
+const archiveDirectMessageThread = (
+  userDirectMessageThreadId: string
+): Promise<Object> => {
+  return db
+    .table('usersDirectMessageThreads')
+    .get(userDirectMessageThreadId)
+    .update(
+      {
+        archivedAt: new Date(),
+        receiveNotifications: false,
```

Comment:
This should **turn notifications off** when a thread is unarchived, right? @mxstbr @brianlovin